source/content/exercise/problem.md

```diff
@@ -0,0 +1,9 @@
+# Exercise
+
+In this exercise, you will work on a LaTeX article with a bibliography. 
+You will learn how to set-up an automated CI/CD pipeline using GitHub Actions to compile the LaTeX document and publish it as a PDF file.
+This setup will ensure that your document is consistently compiled with all the necessary dependencies and configurations.
+
+By the end of this exercise, you will have a practical workflow that enhances your productivity and ensures the quality of your LaTeX documents.
+
+Go to the [LaTeX Article CI/CD Template Repository](https://github.com/matteodelucchi/latex-article-template) and follow the instructions in the README.
```

Comment:
@j-i-l Should I keep it on my profile, or do you wanna fork it to t3d-gmbh? I have no preference, but if necessary, feel free to change the link in this line.

Reply:
I have no preference either. If you don't mind we keep it where it is. 
But maybe you could add me to the repo, so I could edit.